### ISSUE TITLE
Shibboleth did not redirect logout to ADFS

### ISSUE THREAD

**Issue by zeekreey** [DSpace/DSpace#8650]
I'm having trouble with Shibboleth configuration. When logging in through Shibboleth, it redirects the login page to my ADFS and successfully redirected to the home page. Whenever I clicked on logout button, it logout locally instead of redirecting to ADFS logout page. Hope anyone can provide a guidance to this.

**Comment by tdonohue** [comment]
@zeekreey : We don't currently have "Single Logout support" for Shibboleth see this ticket #7799

In other words, if you logout in DSpace, it just logs you out from *DSpace itself*.  You remain logged into Shibboleth, but your DSpace session is ended.

There's also a known bug related to this in #8475 which is waiting on a volunteer.

Overall, this ticket appears to be a duplicate of #7799

**Comment by mwoodiupui** [comment]
Please allow me to note that I consider the current situation a feature.  I think SLO should be a separate operation from logging out of a single service.  (It would be quite reasonable to offer a choice:  logout of DSpace, or logout of DSpace *and* a central authority.)

**Comment by tdonohue** [comment]
Closing as duplicate of #7799 .  This feature doesn't exist yet & needs more discussion.
